<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>818</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Eucl</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>3</width>
      <height>50</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>15</number>
    </property>
    <item>
     <property name="text">
      <string>Point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Point on line</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Point on circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Intersection point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Midpoint of segment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Midpoint of circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Orthogonal projection</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bisector</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Translate with vector</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Perpendicular</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rotation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Make grid</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Segment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Polygon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Linestring</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>190</y>
      <width>391</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>115</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>point</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>115</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>segment</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>3</width>
      <height>50</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Circumscribed circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circle by its radius</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sector</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inscribed circle</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>115</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>circle</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>3</width>
      <height>50</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>115</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>interactions</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Move point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Move canvas</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>170</y>
      <width>381</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-512</number>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>160</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>160</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> + /  -</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="coord_system">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>0</y>
      <width>120</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>coord system</string>
    </property>
    <widget class="QCheckBox" name="grid_show">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>show grid</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="x_axis_show">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>show x axis</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="y_axis_show">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>show y axis</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>3</width>
      <height>50</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>115</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>decorators</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Mark angle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mark right angle</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>0</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip fill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_add_function">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add
function</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkb_autocompile">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>auto-compile</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkb_canvas_always_on">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>canvas objects always on</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_copy_tikzpicture">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>110</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>copy tikzpicture to clipboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_copy_document">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>130</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>copy document to clipboard</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkb_show_pdf">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>110</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>show pdf</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_pdflatex">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>690</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>670</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Messages</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionopen_settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionopen_settings">
   <property name="text">
    <string>open settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
